packages/components/src/components/breadcrumb/breadcrumb-item-label.tsx

```diff
@@ -0,0 +1,28 @@
+import { Component, h, Prop, Element } from '@stencil/core';
+
+@Component({
+  tag: 'ifx-breadcrumb-item-label',
+  styleUrl: 'breadcrumb-item-label.scss',
+  shadow: true,
+})
+export class BreadcrumbItemLabel {
+  @Prop() icon: string;
+  @Prop() url: string;
+  @Prop() target: string = "_self"
+  @Element() el;
+
+
+  render() {
+    return (
+      <a href={this.url} target={this.target} class="breadcrumb-item-label-container">
+        <ifx-icon icon={this.icon}></ifx-icon>
+        <span class="label-wrapper">
+          <slot />
+        </span>
+       <span class="menu-icon-wrapper">
```

Comment:
This icon will always be displayed - also for breadcrumb-items that don't have an dropdown.